routes/web.php

```diff
@@ -520,6 +519,7 @@
         Route::get('version/latest', 'HomeController@latestVersion');
         Route::post('update-latest-version', 'HomeController@updateLatestVersion');
         Route::post('v1/checkUpdatesExpiry', 'HomeController@checkUpdatesExpiry');
+        Route::get('update/lic-code', 'HomeController@updateLicenseCode');
```

Comment:
make it a post request and make url more readable. Something like `refresh-license-code`